setup.py

```diff
@@ -1,17 +1,8 @@
 import setuptools
 
-EXTRAS_REQUIRE = {
-        'geo': [
-            "geopy==2.1.0",
-            "rasterio>=1.2.10,<2.0.0",
-            "keplergl>=0.3.2,<0.4.0",
-            "simplekml>=1.3.6,<2.0.0"
-            ]
-        }
-
 setuptools.setup(
     name="mapreader",
-    version="0.3.3",
+    version="0.3.4",
     description="A computer vision pipeline for the semantic exploration of maps/images at scale",
     author=u"kasra-hosseini",
     #author_email="",
```

Comment:
Do we need to change author + email contact information here to someone who is actively handling dev of MapReader? Either @rwood-97 or @andrewphilipsmith perhaps?

Reply:
I did also wonder about this - I wondered if we should change to something more generic like 'MapReader team' then if we change about it won't matter so much?